samples/CommunityToolkit.Maui.Sample/Pages/Converters/IndexToArrayItemConverterPage.xaml

```diff
@@ -4,12 +4,60 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
     xmlns:mct="clr-namespace:CommunityToolkit.Maui.Converters;assembly=CommunityToolkit.Maui"
     xmlns:pages="clr-namespace:CommunityToolkit.Maui.Sample.Pages"
+    xmlns:vm="clr-namespace:CommunityToolkit.Maui.Sample.ViewModels.Converters"
     x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.IndexToArrayItemConverterPage">
+    <pages:BasePage.BindingContext>
+        <vm:IndexToArrayItemConverterViewModel />
+    </pages:BasePage.BindingContext>
+    <ContentPage.Resources>
+        <mct:IndexToArrayItemConverter x:Key="IndexToArrayItemConverter" />
+        <x:Array x:Key="StringArray" Type="{x:Type x:String}">
+            <x:String>Value 0</x:String>
+            <x:String>Value 1</x:String>
+            <x:String>Value 2</x:String>
+            <x:String>Value 3</x:String>
+            <x:String>Value 4</x:String>
+            <x:String>Value 5</x:String>
+            <x:String>Value 6</x:String>
+            <x:String>Value 7</x:String>
+        </x:Array>
+    </ContentPage.Resources>
     <pages:BasePage.Content>
         <StackLayout>
-            <Label Text="IndexToArrayItemConverter"
-                VerticalOptions="CenterAndExpand" 
-                HorizontalOptions="CenterAndExpand" />
+            <Label Text="The IndexToArrayItemConverter is a converter that allows users to convert a int value binding to an item in an array. The int value being data bound represents the indexer used to access the array. The array is passed in through the ConverterParameter."
+                VerticalOptions="FillAndExpand" 
+                HorizontalOptions="FillAndExpand" />
+                
+                <StackLayout Margin="0,20,0,20">
+                    <Label Text="Change SelectedIndex to change SelectedItem" 
+                           FontAttributes="Italic"/>
+                    <StackLayout Orientation="Horizontal" 
```

Comment:
(Just FYI since we're going to convert this to a `Grid`)
.NET MAUI introduced `VerticalStackLayout` and `HorizontalStackLayout`, so going forward it's a code smell to use `StackLayout Orientation="Horizontal"`